lib/src/barback/dartdevc/workers.dart

```diff
@@ -13,4 +13,8 @@ final analyzerDriver = new BazelWorkerDriver(() => Process.start(
     p.join(sdkDir.path, 'bin', 'dartanalyzer'),
     ['--build-mode', '--persistent_worker']));
 
+/// Manages a shared set of persistent dartdevc workers.
+final dartdevcDriver = new BazelWorkerDriver(() => Process
+    .start(p.join(sdkDir.path, 'bin', 'dartdevc'), ['--persistent_worker']));
```

Comment:
It actually spins up a few of these (4 by default). They are shut down when pub shuts down (we want to keep them hot).

Eventually I think we might want to make the number configurable, the BazelWorkerDriver accepts both a max number of idle workers and a max number of total workers, both default to 4 (same as bazel itself).